src/root/port.c

```diff
@@ -337,6 +345,9 @@ int Port::stricmp(const char *s1, const char *s2)
 
 float Port::strtof(const char *p, char **endp)
 {
+#if _MSC_VER >= 1900
+    return ::strtof(p, endp); // C99 conformant since VS 2015
```

Comment:
Incl. setting errno to ERANGE for underflows. From MSDN:
> For both functions [strtof and wcstof], errno is set to ERANGE if overflow or underflow occurs.
They also support hex strings now, although MSDN doesn't mention it.